content/zh/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -135,7 +135,7 @@ Here are some ideas for how to use init containers:
 
 * 在启动应用容器之前等一段时间，使用类似命令：
 
-        `sleep 60` 
```

Comment:
this should be not changed.

Reply:
![image](https://user-images.githubusercontent.com/23489782/80218921-65c59f80-8674-11ea-8f95-ec58546383fd.png)

Reply:
There should be something wrong with this format

Reply:
see 
![image](https://user-images.githubusercontent.com/19339970/80237002-8ac70c00-868e-11ea-8f60-62237f5e5e43.png)

Reply:
maybe be 
\`\`\`shell
sleep 60
\`\`\`  is much better.